docs/tutorial/first-app.md

```diff
@@ -104,6 +113,14 @@ RewriteCond %{REQUEST_FILENAME} !-f
 RewriteRule . index.php [L]
 ```
 
+nginx does not use `.htaccess` files, so you will need to add the following to your server configuration in the `location` block:
+
+```
+if (!-e $request_filename){
+    rewrite ^(.*)$ /index.php break;
```

Comment:
Sounds good, my goal was just a few nginx 'subtitles' for the existing Apache config in the tutorial.